### ISSUE TITLE
CTRL-C should not invoke the AI debugger

### ISSUE THREAD

**Issue by acote88** [DefangLabs/defang#588]
Repro:
1. Run "defang compose up"
2. CTRL-C to terminate this command since user forgot to use the --verbose flag

Result:
```
! Deployment is not finished. Service(s) might not be running.
? Would you like to debug the deployment with AI?
```

Expected:
1. Since the deployment is not finished as the text says, why is it even asking for AI debugger?
2. CTRL-C needs to terminate the command immediately and not prompt for things

**Comment by lionello** [comment]
Just press `V`!

**Comment by lionello** [comment]
Fixed by #691
